### PR TITLE
Added renew to session info, fixed pom file

### PR DESCRIPTION
**Pull request by weberr13**

### CODE REVIEW

File: consul-client-0.9.pom [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      <version>3.0</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.5.3</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
```
Comment (weberr13) [inline consul-client-0.9.pom]: This was not present on http://dl.bintray.com/orbitz/consul-client/com/orbitz/consul/consul-client/0.9/#consul-client-0.9.pom and it caused issues in runtime

File: src/main/java/com/orbitz/consul/SessionClient.java [hunk 2]
```diff
@@ -4,149 +4,175 @@
...
+         target = webTarget.queryParam("dc", dc);
+      }
+
+      session = target.path("create").request().put(Entity.entity(value,
+              MediaType.APPLICATION_JSON_TYPE), new GenericType<Map<String, String>>() {
+              });
+
+      return session != null ? Optional.of(session.get("ID")) : Optional.<String>absent();
+   }
+
```
Comment (weberr13) [inline src/main/java/com/orbitz/consul/SessionClient.java]: new stuff starts here

File: src/main/java/com/orbitz/consul/SessionClient.java [hunk 3]
```diff
@@ -4,149 +4,175 @@
...
+      if (dc != null) {
+         target = webTarget.queryParam("dc", dc);
+      }
+
+      Response session = target.path("renew").path(sessionId).request().put(Entity.entity("{}",
+              MediaType.APPLICATION_JSON_TYPE));
+      return session.hasEntity();
+
+   }
+
```
Comment (weberr13) [inline src/main/java/com/orbitz/consul/SessionClient.java]: new stuff ends here.

**Comment by rickfast** [comment]
the java changes look okay to me.  the POM file is generated by the gradle build when I cut a release.  I just looked in Bintray, and the 0.9 POM is definitely there.

https://bintray.com/artifact/download/orbitz/consul-client/com/orbitz/consul/consul-client/0.9/consul-client-0.9.pom

Were you trying to resolve the dependency via Maven or Gradle?

**Comment by weberr13** [comment]
I was using Maven.  I can try again with the url you gave.  I'll clean this request up and remove the pom files from it so you can look at just getting the java changes in.  Just a sec.

**Comment by weberr13** [comment]
OK, pom files are removed, if you want it you should be able to merge.

**Comment by rickfast** [comment]
i'll try with maven today and see if i can figure out what the problem is.  we've only used gradle to resolve it and haven't had any issues.

**Pull request merged**